spec/views/posts/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe "posts/edit", type: :view do
+  before(:each) do
+    @post = assign(:post, Post.create!(
+      content: "MyText",
+      grass_tags: "MyString",
+      only_followers: true
+    ))
+  end
+
+  it "renders the edit post form" do
+    render
+
+    assert_select "form[action=?][method=?]", post_path(@post), "post" do
+
+      assert_select "textarea[name=?]", "post[content]"
+
+      assert_select "input[name=?]", "post[grass_tags]"
+    end
+  end
+end
```

Comment:
Most of these views tests are original to the scaffolding, but I've made slight changes here, like with taking out the assertion that `only_followers` would be viewable